test/evaluation/markForSubmissionEvaluationHandler.spec.ts

```diff
@@ -0,0 +1,201 @@
+import fs from 'fs';
```

Comment:
[Optional] (suggestion) For some of the examples, if possible, please create files instead of keeping them in code.